synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -303,21 +317,51 @@ Widget_Timeslider::on_draw(const Cairo::RefPtr<Cairo::Context> &cr)
 				double x1 = time_plot_data->get_double_pixel_t_coord(bounds[i][1]);
 				double w = x1 - x0;
 
+				double boundary_dimension=8;
+				double background_adjust;
+
+				if(i==0)
+					background_adjust=0;
+				else
+					background_adjust=8;
+
 				cr->save();
-				cr->rectangle(x0, 0.0, w, (double)get_height());
+				cr->rectangle(x0 + background_adjust, 0.0, w-8, (double)get_height());
 				cr->clip();
 				cr->translate(offset, 0.0);
 				cr->set_source(play_bounds_pattern);
 				cr->paint();
 				cr->restore();
 
+				Glib::RefPtr<Gdk::Pixbuf> icon;
```

Comment:
I changed the icon just to show the problem:
Icon:
![Screenshot_59](https://user-images.githubusercontent.com/5604544/181670279-3c7cce25-8b02-413d-ad16-61119f35bc33.png)

Problem:
![Screenshot_58](https://user-images.githubusercontent.com/5604544/181670339-9a998002-f1e7-435f-bb11-6a229338e641.png)

![Screenshot_60](https://user-images.githubusercontent.com/5604544/181670481-ade8b4bf-4a5c-4485-b078-10e3845bad90.png)

As you can see, it overlaps with borders.
My suggestion is to first draw the borders and then draw a handle over them.

Reply:
ok sounds good will do.